onnx/backend/test/runner/__init__.py

```diff
@@ -141,14 +146,14 @@ def tests(self):
         return tests
 
     @staticmethod
-    def _assert_similar_outputs(ref_outputs, outputs):
+    def _assert_similar_outputs(ref_outputs, outputs, rtol, atol):
```

Comment:
you probably want to set the default values, in case you have a lot node tests, do you really want to manually set them one by one. Check line 296

Reply:
The default values are still there, they just moved to the `_tolerance` function.